yb-voyager/cmd/archiveChangesCommand.go

```diff
@@ -203,13 +203,13 @@ func (m *EventSegmentCopier) copySegmentFile(segment Segment, segmentNewPath str
 }
 
 func (m *EventSegmentCopier) Run() error {
-	importCount, err := m.getImportCount()
-	if err != nil {
-		return fmt.Errorf("getimport count: %v", err)
-	}
-	log.Infof("Import count: %d", importCount)
 
 	for {
+		importCount, err := m.getImportCount()
+		if err != nil {
+			return fmt.Errorf("getimport count: %v", err)
+		}
+		log.Infof("Import count: %d", importCount)
```

Comment:
This message will pollute the log file. `PrintAndLog` only when the value of `importCount` changes from the previous iteration.